_python/wiki_scrapping.py

```diff
@@ -42,36 +38,30 @@ def parse_tables(self,soup,table_index):
             row_data = self.parse_row(row)
             data.append(row_data)
             
-        self.df = pd.DataFrame(data,columns=headers)
+        self.df = pd.DataFrame(data, columns=headers)
         return self.df
     
-    def extract_headers(self,table):
+    def extract_headers(self, table):
         headers = [header.get_text(strip=True) for header in table.find_all('th')]
-        logging.info(f'Parsed headers:{headers}')
+        logging.info(f'Parsed headers: {headers}')
         return headers
-
-    def parse_row(self,row):
+    @staticmethod
+    def parse_row(row):
         row_data = row.find_all('td')
         each_row = [data.text.strip() for data in row_data]
         return each_row
-    
 
-    def pd_Clean(self):
+    def clean_data(self):
        logging.info('Cleaning data')
```

Comment:
Something wrong with indentation here(assuming it is 2 spaces instead of 4)

Reply:
I don't think it was fixed, please check other lines as well
<img width="515" alt="Screenshot 2024-10-26 at 00 43 07" src="https://github.com/user-attachments/assets/a57bd430-3bd3-437c-897c-e910e673b889">

Reply:
Should be fixed by now, sorry